packages/terra-core-docs/src/terra-dev-site/doc/divider/About.1.doc.mdx

```diff
@@ -42,8 +42,8 @@ import Divider from 'terra-divider';
 
 ## Examples
 <DividerExample title="Default" />
-<DividerWithText title="Custom Text" />
-<DividerWithWrappingText title="Custom Text" />
+<DividerWithText title="Divider With Custom Heading Level" />
```

Comment:
Need to update this example [DividerWithText.jsx](https://github.com/cerner/terra-core/blob/divider_subcomp/packages/terra-core-docs/src/terra-dev-site/doc/divider/example/DividerWithText.jsx) to also include the new level prop `level={3}`

\
This updated title sounds a little clearer:
```suggestion
<DividerWithText title="Section Divider with Heading Text" />
```

Reply:
Also, none of the WDIO tests -- [DividerWithText.test.jsx](https://github.com/cerner/terra-core/blob/divider_subcomp/packages/terra-core-docs/src/terra-dev-site/test/divider/DividerWithText.test.jsx) and [DividerWithWrappingText.test.jsx](https://github.com/cerner/terra-core/blob/divider_subcomp/packages/terra-core-docs/src/terra-dev-site/test/divider/DividerWithWrappingText.test.jsx) -- have been updated yet to capture the new level to make sure appearances look correct. Rather than adding new tests, you could simplify by just adding more to the existing tests. 

\
For example for [DividerWithText.test.jsx#L8-L14](https://github.com/cerner/terra-core/blob/19ae0bf6518e9c5c93cd89816163da3aa57ba9ec/packages/terra-core-docs/src/terra-dev-site/test/divider/DividerWithText.test.jsx#L8-L14), changing it to:
```
export default () => (
  <div>
    <div className={cx('separator')} />
    <Divider id="divider" text="Custom Divider Text" />
    <div className={cx('separator')} />
    <Divider id="divider" text="Section Divider H1 Text" level={1} />
    <div className={cx('separator')} />
    <Divider id="divider" text="Section Divider H2 Text" level={2} />
    <div className={cx('separator')} />
    <Divider id="divider" text="Section Divider H3 Text" level={3} />
    <div className={cx('separator')} />
    <Divider id="divider" text="Section Divider H4 Text" level={4} />
    <div className={cx('separator')} />
    <Divider id="divider" text="Section Divider H5 Text" level={5} />
    <div className={cx('separator')} />
    <Divider id="divider" text="Section Divider H6 Text" level={6} />
    <div className={cx('separator')} />
  </div>
);
```

\
And for [DividerWithWrappingText.test.jsx#L10-L16](https://github.com/cerner/terra-core/blob/19ae0bf6518e9c5c93cd89816163da3aa57ba9ec/packages/terra-core-docs/src/terra-dev-site/test/divider/DividerWithWrappingText.test.jsx#L10-L16), changing it to:
```
export default () => (
  <div>
    <div className={cx('separator')} />
    <Divider id="divider" text={content} />
    <div className={cx('separator')} />
    <Divider id="divider" text={`H1: ${content}`} level={1} />
    <div className={cx('separator')} />
    <Divider id="divider" text={`H2: ${content}`} level={2} />
    <div className={cx('separator')} />
    <Divider id="divider" text={`H3: ${content}`} level={3} />
    <div className={cx('separator')} />
    <Divider id="divider" text={`H4: ${content}`} level={4} />
    <div className={cx('separator')} />
    <Divider id="divider" text={`H5: ${content}`} level={5} />
    <div className={cx('separator')} />
    <Divider id="divider" text={`H6: ${content}`} level={6} />
    <div className={cx('separator')} />
  </div>
);
```